packages/backend/server/src/server.ts

```diff
@@ -60,11 +60,10 @@ export async function run() {
   app.useWebSocketAdapter(adapter);
 
   const url = app.get(URLHelper);
-  const listeningHost = '0.0.0.0';
 
-  await app.listen(config.server.port, listeningHost);
+  await app.listen(config.server.port, config.server.host);
```

Comment:
After supporting multi-host deployment, this host configuration cannot currently be used directly as the listen host https://github.com/toeverything/AFFiNE/pull/12950